pkg/email/templates/shared_style.html

```diff
@@ -135,6 +135,9 @@
       line-height: normal;
       color: #71767A;
     }
+    .mention {
+      color: #005EA2;

```

Comment:
if desired in the future, this could be expanded to make the mentions have additional behavior